modules/Todo.js

```diff
@@ -0,0 +1,113 @@
+import Checkbox from '../src/img/maximize-button.png';
+import VerticalMenu from '../src/img/vertical-menu.png';
+import TrashCan from '../src/img/trash-can.png';
+
+let todos;
+
+if (localStorage.getItem('todos')) {
+  todos = JSON.parse(localStorage.getItem('todos'));
+} else {
+  todos = [];
+}
+
+export const updateTodoIndex = () => {
+  todos.forEach((todo, i) => {
+    todo.index = i + 1;
+  });
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
+
+export class Todos {
+  constructor(description, completed, index) {
+    // this.todos = [];
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+    // this.todos = [{ description, completed, index }];
```

Comment:
- [ ] Kindly do not leave commented out code. They can make your code confusing and they don't add functionality to your code.